Add refresh token documentation specific to the OIE SDK and widget (#2291)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-refresh-tokens/main/nodeexpress/refreshusingthesdk.md

## Refresh using the Javascript SDK

> The Javascript SDK has built-in support for auto renewing of access and

Reviewer: ```suggestion
The Javascript SDK has built-in support for the auto-renewing of access and
```

> id tokens. This behavior is enabled by default, but can be disabled by

Reviewer: ```suggestion
ID tokens. This behavior is enabled by default, but you can disable it by
```

setting the `tokenManager's` `autoRenew` property to `false`.

```javascript
const authClient = new OktaAuth({
    tokenManager: {
      autoRenew: false
    }
  });
```

With this property set to `false`, you can still manually refresh tokens by

> subscribing to the `tokenManager's` `expired` event and calling it's

Reviewer: I am not a big fan of writing these in the possessive. I don't think it really needs to be:
 
```suggestion
subscribing to the `tokenManager` `expired` event and calling its
```

`renew` method.

```javascript
authClient.tokenManager.on('expired', function (key, expiredToken) {
  //do something
});
```

> For more information about these methods and events, see the links below that

Reviewer: ```suggestion
See the following links to the SDK repository [readme](https://github.com/okta/okta-auth-js#readme) for more information about these methods and events.
```

> refer to SDK's repository [readme](https://github.com/okta/okta-auth-js#readme).

Reviewer: ```suggestion

```

> * [`tokenManager's` `autoRenew` property](https://github.com//okta/okta-auth-js#autorenew)

Reviewer: ```suggestion
* [`tokenManager` `autoRenew` property](https://github.com//okta/okta-auth-js#autorenew)
```

> * [`tokenManager's` `expired` event](https://github.com/okta/okta-auth-js#tokenmanageronevent-callback-context)

Reviewer: ```suggestion
* [`tokenManager` `expired` event](https://github.com/okta/okta-auth-js#tokenmanageronevent-callback-context)
```

> * [`tokenManager's` `renew` method](https://github.com/okta/okta-auth-js#tokenmanagerrenewkey)

Reviewer: ```suggestion
* [`tokenManager` `renew` method](https://github.com/okta/okta-auth-js#tokenmanagerrenewkey)
```

> **Note:** The direct approach using the

> [OAuth token endpoint](#refresh-using-the-oauth-token-endpoint) is also supported to

Reviewer: ```suggestion
[OAuth token endpoint](#refresh-using-the-oauth-token-endpoint) is also supported for
```

## Option 1: Refresh using the Javascript SDK

The Javascript SDK has built-in support for the auto-renewing of access and

> id tokens. This behavior is enabled by default, but you can disabled it by

Reviewer: ```suggestion
ID tokens. This behavior is enabled by default, but you can disabled it by
```

> ID tokens. This behavior is enabled by default, but you can disabled it by

Reviewer: ```suggestion
ID tokens. This behavior is enabled by default, but you can disable it by
```
